var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -377,7 +377,7 @@ def common_config_args(self):
         if self.spec.variants['freetype'].value == 'spack':
             config_args.extend([
                 '-system-freetype',
-                '-I{0}/freetype2'.format(self.spec['freetype'].prefix.include)
+                *self.spec['freetype'].headers.include_flags.split()
```

Comment:
Maybe I was wrong? Seems to be a syntax error now. This might be a Python 2 vs. 3 thing.

Reply:
Probably, I only tested it with Python 3. I have reverted this.